test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -217,4 +217,8 @@ export default class BasePageObject {
 
     return elCount === 1;
   }
+
+  hasText(text) {
+    return this.form.getTextElement(text).exists;
+  }
```

Comment:
put in the BasePageObject so it can be used by all tests that need to check if text exists